src/src.pro

```diff
@@ -0,0 +1,199 @@
+######################################################################
```

Comment:
Hmmmm, this whole section doesn't seem correct.

Not sure why it's included, as it's not specific to Romanian? :smile: